src/commands/service/create.decorator.json

```diff
@@ -77,6 +77,16 @@
       "arg": "--path-prefix-major-version <path-prefix-major-version>",
       "description": "Version to be used in the path prefix; will be used to configure Traefik2 IngressRoutes. ie. 'v1' will result in a path prefix of '/v1/servicename",
       "defaultValue": ""
+    },
+    {
+      "arg": "--service-build-vg <name of variable group for service build>",
+      "description": "List of variable groups to be linked to service pipeline",
```

Comment:
If multiple, help should indicate how they are separated.

Reply:
There is an example in the project-service-management-guide.md.

Reply:
but I can update the help to include this as well.